.github/workflows/podman-aws-e2e.yaml

```diff
@@ -0,0 +1,62 @@
+# Runs Azure tests when pull request opened, repopened or synchronized
+name: Podman E2E Tests - Build AWS AMI
+on:
+  workflow_dispatch:
+  workflow_call:
+
+permissions:
+  contents: read
+  id-token: write
+
+jobs:
+  rune2e:
+    runs-on: ubuntu-latest
```

Comment:
Do we want to use the self hosted runners here?

Reply:
this just has podman on it ?

Reply:
This runner has both podman and docker on it. the selfhosted runners don't have podman installed. https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md